src/patternfly/components/Login/docs/code.md

```diff
@@ -12,11 +12,12 @@
 | `.pf-c-login__main-header .pf-c-dropdown` | `<div>` |  Creates a language selector dropdown in the main header area. |
 | `.pf-c-login__main-body` | `<div>` |  Creates the body of the main area. |
 | `.pf-c-login__main-body .pf-c-form` | `<form>` |  Creates the login form in the main body area. |
+| `.pf-c-login__main-body .pf-c-form .pf-c-form-helper-text.pf-m-error` | `<form>` |  Creates the error text above the form fields, this should always be present on the page with `.pf-m-hidden` applied when it is not active. |
 | `.pf-c-login__main-footer` | `<footer>` |  Creates the footer of the main area. |
-| `.pf-c-login__main-footer .pf-c-title` | `<h1>,<h2>,<h3>,<h4>,<h5>,<h6>` |  Creates a title in the main footer area. |
 | `.pf-c-login__main-footer .pf-c-list` | `<ul>` |  Creates a list of links in the main footer area. |
-| `.pf-c-login__main-footer-link` | `<a>` |  Creates links in the main footer area. |
-| `.pf-c-login__main-footer-link-text` | `<span>` |  Creates text in the in the main footer links. |
+| `.pf-c-login__main-footer-links` | `<ul>` |  Creates links in the main footer area. |
```

Comment:
Could we change this to:
`Creates an unordered list of links in the main footer`